src/modules/browse.js

```diff
@@ -42,6 +43,11 @@ function createQueryParams(parameters, userParameters, options) {
       queryParams.page = page;
     }
 
+    // Pull offset from parameters
+    if (!helpers.isNil(offset)) {
```

Comment:
What's the thinking on `isNil` vs `isInteger`?

Reply:
At first I just saw isNil used in the other cases in the code so I just used it as well 😅 But after looking we are sending all the params to the qs.stringify function anyway so a value like '1' which is not an integer should also be accepted. Am I right?

Reply:
Good point about the rest of the checks using `isNil` - thanks for the clarification. I think we're good to go!